### DISCUSSION TITLE
Similar to how input works, outputs should also have a label

### DISCUSSION THREAD

**Discussion by WASasquatch** [comfyanonymous/ComfyUI#459] [Ideas]
I stand with the coalition for Labels on Outputs. We stand united. Although it's just me. Turns out I have to deliver the flyers myself when it's just me.

**Comment by comfyanonymous** [comment]
That's implemented: https://github.com/comfyanonymous/ComfyUI/blob/master/custom_nodes/example_node.py.example#L66

**Reply by WASasquatch** [reply]
Oh awesome, that wasn't in the examples when I started.

**Comment by omar92** [comment]
lol we need docs XD

**Reply by WASasquatch** [reply]
Making use of the wiki would be great. Contributors could even document their PRs once they are a contributor. 

All the stuff on random other repos could be on the wiki too, and downloadable like any other software docu/examples.
